mlos_bench/mlos_bench/launcher.py

```diff
@@ -91,12 +92,15 @@ def __init__(self, description: str, long_text: str = ""):
                          " Run `mlos_bench --help` and consult `README.md` for more info.")
         self.root_env_config = self._config_loader.resolve_path(env_path)
 
-        tunable_groups = TunableGroups()    # base tunable groups that all others get build on
         self.environment: Environment = self._config_loader.load_environment(
-            self.root_env_config, tunable_groups, self.global_config, service=self._parent_service)
+            self.root_env_config, TunableGroups(), self.global_config, service=self._parent_service)
+
+        # NOTE: Init tunable values *after* the Environment, but *before* the Optimizer
```

Comment:
Why's that?

Reply:
a line just above this one is `tunable_groups = TunableGroups()` and we don't use that variable anywhere else

Reply:
I saw that, I was just curious if there was something about that logic that's important to work that way.  Do we manipulate the params in the Environment somehow?